### ISSUE TITLE
MQTT over WebSockets

### ISSUE THREAD

**Issue by johnjore** [eclipse/paho.mqtt.m2mqtt#7]
Hi!

Are there any plans to add websocket support ?

JJ

**Comment by uvzubovs** [comment]
Indeed,  very much needed feature. Java/android client has had it for a while now. Thanks in advance!

**Comment by Aeefire** [comment]
Is there any update on this?

Are there any special hindrances which could cause issues ?
